go-chaos/cmd/cluster.go

```diff
@@ -45,11 +46,88 @@ func AddClusterCommands(rootCmd *cobra.Command, flags *Flags) {
 			return waitForChange(flags)
 		},
 	}
+	var scaleCommand = &cobra.Command{
+		Use:   "scale",
+		Short: "Scales the cluster to the given size",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return scaleCluster(flags)
+		},
+	}
 
 	rootCmd.AddCommand(clusterCommand)
 	clusterCommand.AddCommand(statusCommand)
 	clusterCommand.AddCommand(waitCommand)
 	waitCommand.Flags().IntVar(&flags.changeId, "changeId", 0, "The id of the change to wait for")
+	clusterCommand.AddCommand(scaleCommand)
+	scaleCommand.Flags().IntVar(&flags.brokers, "brokers", 0, "The amount of brokers to scale to")
```

Comment:
❓ Is zero really the default you want to have here? I would recommend it is either three ( our standard setting) or one, or something you would normally use in experiments.

Reply:
Oh yeah good point :+1:

Reply:
It's a required flag though so the default of 0 is unused anyway. I'd leave it as is/